src/ctree-app/ctree-landing.html

```diff
@@ -278,30 +407,32 @@
   </template>
 
   <script>
-    class CTreeLanding extends Polymer.mixinBehaviors([Polymer.IronScrollTargetBehavior], Polymer.Element) {
-
-      static get is() { return 'ctree-landing'; }
+    class CTreeLanding extends Polymer.mixinBehaviors(
+      [Polymer.IronScrollTargetBehavior],
+      Polymer.Element
+    ) {
+      static get is() {
+        return "ctree-landing";
+      }
 
       static get properties() {
         return {
           rootPath: {
             type: String,
-            value: '',
+            value: ""
           },
 
           section: String,
 
           _suppressInferSection: {
             type: Boolean,
-            value: false,
-          },
```

Comment:
I'll often leave the comment at the end so it's easier to add additional entries.  I'm not opposed to removing it, but it's not a big deal because it works fine either way.